OpenRA.Mods.Common/Traits/GainsExperience.cs

```diff
@@ -20,7 +19,6 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("This actor's experience increases when it has killed a GivesExperience actor.")]
 	public class GainsExperienceInfo : ITraitInfo, Requires<ValuedInfo>, Requires<UpgradeManagerInfo>
 	{
-		[FieldLoader.LoadUsing("LoadUpgrades", true)]
```

Comment:
The `true` indicates that this is a required field.  You will need to add the appropriate attribute to not change behavior here.